### PR TITLE
common: additional telemetry improvements for 9xr pro

### PR DESCRIPTION
**Pull request by larsks**
This improves the docs for telemetry on the Turnigy 9XR Pro in several
ways:

- consistent lighting, sizing and positioning of screenshots
- correct information regarding SERIAL#_PROTOCOL (ErSky9x supports the
  passthrough telemetry protocol)
- information about configuring custom telemetry views

### CODE REVIEW

File: common/source/docs/common-frsky-repurposed.rst [hunk 1]
```diff
@@ -63,33 +97,7 @@ When configured to display “GPS,” the custom telemetry screen will show long
...
-| Tmp1                                | Control/flight mode                                           |
-+-------------------------------------+---------------------------------------------------------------+
-| Tmp2                                | GPS status and number of satellites (as num_sats*10 + status) |
-+-------------------------------------+---------------------------------------------------------------+
+Configuringation with ErSky9x
+=============================
 
-If you installed FrSky sensors on your vehicle, other messages from these may also appear during discovery.
\ No newline at end of file
+For information on how to configure your ErSky9x transmitter (such as a Turnigy 9XR Pro) for FrSky telemetry, please go :ref:`here <common-frsky-telemetry>`.
```
Comment (khancyr) [inline common/source/docs/common-frsky-repurposed.rst]: typo ErSky9x -> FrSky9x

Comment (larsks) [inline common/source/docs/common-frsky-repurposed.rst]: Thanks for reading.  That's not a typo. "ErSky9x" is the name of the firmware with telemetry support. You can read more about it, e.g., http://openrcforums.com/forum/.

**Comment by larsks** [comment]
ping @magicrub, is this okay? Thanks!

**Comment by rmackay9** [comment]
Merged, thanks!

**Pull request merged**